sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleFacadeEjb.java

```diff
@@ -968,6 +973,130 @@ public void deleteAllSamples(List<String> sampleUuids) {
 		return sampleService.getNewTestResultCountByResultType(caseIds);
 	}
 
+	@Override
+	public Map<SampleCountType, Long> getSampleCount(
```

Comment:
This method results in 11 database calls being executed whenever someone accesses the new dashboard; I would like you to write a dedicated method instead that is only called once, does a multiselect on the fields that you need (i.e. specimen condition, pathogen test result, shipped, received), and then builds the map based on the returned data. This method of course still needs to incorporate the user filter (like the count method does).